runtime/descriptor.go

```diff
@@ -117,5 +119,31 @@ func makeStructFieldDescriptor(t *Type, fieldName, propertyName string) *Object
 		}
 		return ret, raised
 	}
+	if writtable {
+		setterFunc := func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+			ret := None
+			var raised *BaseException
+			if raised = checkFunctionArgs(f, fieldName, args, ObjectType, ObjectType); raised == nil {
```

Comment:
I had followed the style of ``getterFunc()`` above, but can change mine or both to what you are asking. Please confirm.